jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -24,6 +24,16 @@ defaults
     timeout http-request    <%= p("ha_proxy.request_timeout").to_i    * 1000 %>ms
     timeout queue           <%= p("ha_proxy.queue_timeout").to_i      * 1000 %>ms
 
+  <% if p("ha_proxy.stats_enable") %>
+  listen stats :9000
```

Comment:
Should we lock this down so only certain IP ranges can access the stats, and only certain interfaces listen to relay stats? Also, what happens in the unlikely event that some app behind haproxy requires the /haproxy_stats URL?